script.js

```diff
@@ -0,0 +1,63 @@
+// Local Storage Starts Here
+let books = [];
+
+books = JSON.parse(window.localStorage.getItem('books'));
+if (!books) {
+  books = [];
+}
+
+function saveToLocalStorage() {
+  window.localStorage.setItem('books', JSON.stringify(books));
+}
+// Local Storage Ends Here
+
+// Display Books in List starts Here
+function displayBooks() {
+  const booksContainer = document.getElementById('books-container');
+  booksContainer.innerHTML = '';
+  books.forEach((book, index) => {
+    const bookItem = document.createElement('li');
+    bookItem.innerHTML = `<p>${book.title}</p>
+    <p>${book.author}</p>
+    <button>Remove</button>
+    <hr>`;
+
+    booksContainer.appendChild(bookItem);
+    booksContainer.childNodes[index].childNodes[4].onclick = () => {
+      const newBooks = books.filter((book, innerIndex) => index !== innerIndex);
+      books = newBooks;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20remove%20a%20book%20from%20the%20collection%20(hint%3A%20you%20can%20use%20the%20array%20filter()%20method).), Please create a separate function for removing a book like you did with adding a book.